src/test/groovy/org/prebid/server/functional/util/AllureReporter.groovy

```diff
@@ -253,4 +272,46 @@ class AllureReporter extends AbstractRunListener implements IGlobalExtension {
         lifecycle.stopTestCase(uuid)
         lifecycle.writeTestCase(uuid)
     }
+
+
+    private void writeBlocks(FeatureInfo feature, IterationInfo iteration) {
+        def blockText = StringUtils.EMPTY
+        for (BlockInfo block : feature.blocks) {
+            if (!isEmptyOrContainsOnlyEmptyStrings(block.texts)) {
+                def size = block.texts.size()
+                for (int i = 0; i < size; i++) {
+                    if (iteration != null) {
+                        blockText = generationParams(block.texts[i], feature.dataVariables, iteration)
+                    }
+                    def kind = i == 0 ? block.kind.name() : "and"
```

Comment:
can you please extract that "and" to a class level constant. For me it is a little unclear why exactly 'and' is used here when block kind name is not present. Var name will give me more info to understand